Umidigi/A13_Pro_Max_5G/res/xml/power_profile.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="utf-8"?>
+<device name="Android">
+    <item name="ambient.on">0.1</item>
+    <item name="screen.on">0.1</item>
```

Comment:
From the vendor overlay
![image](https://user-images.githubusercontent.com/949967/214030545-9885fe07-96a1-420a-b165-a225e304dae6.png)

Reply:
FYI this is consistent with other Umidigi overlays in this very repo - https://github.com/phhusson/vendor_hardware_overlay/blob/pie/Umidigi/A5pro/res/xml/power_profile.xml

Reply:
well that means you didn't take correct overlay (or you shouldn't take the overlay but framework-res), or umidigi'spower profile is broken.
And ok i'll remove the A5  Pro one

Reply:
I don't think removing it is the correct solution here - if you run a notepad++ scan on all the power profile files, there are many occurrences of these values across different vendors so to me this looks like the power profile submitted in this PR and obtained directly from the stock ROM vendor overlay is correct:

![image](https://user-images.githubusercontent.com/949967/214162183-9b1941a5-df19-45b7-8305-765bc75eaa93.png)

Reply:
There isn't any smartphone screen that consumes 0.1mA when on.

I'll improve the tests so that I stop merging overlays with such stupid profiles, thanks for bringing that up

Reply:
In that case, do you know of how I would go about obtaining the correct values?

Reply:
do `grep power_profile -r /system/frameworks /vendor/overlay /product/overlay /odm/overlay`, and decompile all the APKs that are listed by that command, and see if there is one with values that make more sense.

Reply:
None of the matching files have values that make any more sense - the one in /vendor/overlay is what I took the values from, /product/overlay has the same values

![image](https://user-images.githubusercontent.com/949967/214174325-8e6fffae-f912-4f36-a81f-c9cfc14a1b24.png)

Reply:
what about the one in system/framework/framework-res.apk?

Reply:
framework-res.apk contains these files:

power_profile.xml
power_profile_test.xml

power_profile.xml has the same values but without additional CPU clusters and an incorrect battery capacity - I think this is the aosp template one:
![image](https://user-images.githubusercontent.com/949967/214430547-c476ddd5-0b2b-4a7c-9558-9d0030394720.png)

power_profile_test.xml has what seems like more sensible values but I'm not sure if this file is used, and it has an incorrect battery capacity:
![image](https://user-images.githubusercontent.com/949967/214430832-09e705d7-60c6-4e73-b5d5-56c4515bd1cc.png)

Reply:
K great, let's just stick to the least broken power_profile of the ROM (no the _test is not ok)
The one in /product/overlay/xxxx has the correct battery capacity or did you have to change?
If you didn't have to change it, let's take that.
All measurements of power consumption will be totally broken, but i guess it won't be more broken that stock...

Reply:
The one in /product/overlay had an incorrect battery capacity, the original one submitted in the PR (/vendor/overlay) had the correct one, I didn't have to change it

Reply:
Would you be able to tell whether stock ROM reports non-zero power consumption from the display?

Reply:
I'm not sure how to do that